### PR TITLE
Adding note about crayons design system in forem docs

### PR DESCRIPTION
**Pull request by Rafi993**
## What type of PR is this?

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [x] Documentation Update

## Description
Adding note about Crayon to forem docs

## Related Tickets & Documents

N/A

## QA Instructions, Screenshots, Recordings
1. Run documentation locally by running `yarn gitdocs serve`
2. Visit http://localhost:8000/frontend/styles/

### UI accessibility concerns?

N/A

## Added tests?

- [ ] Yes
- [x] No, and this is why:  This is a documentation update
- [ ] I need help with writing tests

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

- [ ] I've updated the [Developer Docs](https://docs.forem.com) and/or
      [Admin Guide](https://forem.gitbook.io/forem-admin-guide/), or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
- [ ] I've updated the README or added inline documentation
- [ ] I will share this change in a [Changelog](https://forem.dev/t/changelog)
      or in a [forem.dev](http://forem.dev) post
- [ ] I will share this change internally with the appropriate teams
- [ ] I'm not sure how best to communicate this change and need help
- [ ] This change does not need to be communicated, and this is why not: _please
      replace this line with details on why this change doesn't need to be
      shared_

### CODE REVIEW

File: docs/frontend/styles.md [hunk 1]
```diff
@@ -23,3 +23,16 @@ assets in Rails.
 
 For more about branding, theming or design in general in regards to Forem, refer
 to the [Design Guide](/design) documentation.
+
+## Crayons
+
+Crayons is the design system used by Forem. There is a
```
Comment (citizen428) [inline docs/frontend/styles.md:29]: Maybe we can reword this a little bit?

> [Crayons](https://storybook.forem.com/) is the design system used by Forem. A [storybook](https://storybook.js.org/) listing the various elements is available at [https://storybook.forem.com/](https://storybook.forem.com/).
> 
> You can also run it locally with the following command:
> 
> ```
> $ yarn storybook
> ```

**Comment by citizen428** [comment]
Thanks for the contribution @Rafi993, as always it's much appreciated! 😃 I restarted the build for you as there's nothing in this PR that can fail,

File: docs/frontend/styles.md (hunk 1 above)
Comment (citizen428) [inline docs/frontend/styles.md:29]: Maybe we can reword this a little bit?

```
[Crayons]
```

Comment (Rafi993) [reply docs/frontend/styles.md:29]: Let me do that

File: docs/frontend/styles.md [hunk 2]
```diff
@@ -23,3 +23,16 @@ assets in Rails.
...
+[storybook](https://storybook.js.org/) for crayon that has different elements of
```
Comment (ludwiczakpawel) [inline docs/frontend/styles.md:30]:
```suggestion
[storybook](https://storybook.js.org/) for Crayons that has different elements of
```

**Review by ludwiczakpawel — APPROVED** [review]
Little typo but otherwise looks good!

File: docs/frontend/styles.md [hunk 3]
```diff
@@ -23,3 +23,16 @@ assets in Rails.
...
+the design system. You can access hosted version of the storybook in
```
Comment (jacobherrington) [inline docs/frontend/styles.md:31]:
```suggestion
the design system. You can access hosted version of the storybook at
```

File: docs/frontend/styles.md [hunk 4]
```diff
@@ -23,3 +23,16 @@ assets in Rails.
...
+https://storybook.forem.com/
+
+or you can run the storybook locally by running the command
```
Comment (jacobherrington) [inline docs/frontend/styles.md:34]:
```suggestion
or you can run the storybook locally:
```

**Review by jacobherrington — COMMENTED** [review]
Good idea! We definitely need this written down in our docs! Thanks for making this PR. I had a couple of suggestions :+1:

File: docs/frontend/styles.md (hunk 1 above)
Comment (Rafi993) [reply docs/frontend/styles.md]: I have fixed the wording now :)

**Review by citizen428 — APPROVED** [review]

**Review by jacobherrington — APPROVED** [review]
Thanks for adding this super useful note! :+1:

**Comment by cmgorton** [comment]
Thanks for another great contribution @Rafi993 🎉

**Pull request merged**